README.md

```diff
@@ -0,0 +1,30 @@
+# osd-network-verifier
+
+A cli and set of libraries that validates the preconfigured networking components for some osd options.
+
+## Overview
+
+osd-network-verifier can be used prior to the installation of osd/rosa clusters to ensure the pre-requirements are valid for various network options.
+
+### Workflow for egress
+
+* it creates an instance in the target vpc/subnet and wait till the instance gets ready
+* when the instance is ready, an `userdata` script is run against the instance to install appropriate packages, primarily docker
+* the validation is run against the vpc/subnet as containerized https://github.com/openshift/osd-network-verifier/tree/main/build 
+* the output is collected by the cli and written to output
+
+### Subcommands
+
+Take a look at https://github.com/openshift/osd-network-verifier/tree/main/cmd
+
+### Examples
+
+```shel
+AWS_ACCESS_KEY_ID=<redacted> AWS_SECRET_ACCESS_KEY=<redacted> ./osd-network-verifier egress --subnet-id subnet-0ccetestsubnet1864
```

Comment:
https://github.com/openshift/osd-network-verifier/pull/5 will also require the `--image-id=ami-0df9a9ade3c65a1c7` option. I can either add that in that MR or you can add it here, guess it depends really which merges first.